### ISSUE TITLE
Stripe Pro Payment Gateway missing dates in recent changelog updates

### ISSUE THREAD

**Issue by brianleejackson** [awesomemotive/easy-digital-downloads#9649]
### Expected behavior
The changelog for EDD Stripe Pro Payment Gateway has been missing the date in the changelog for the last couple of dates. As you can expect, the dates are helpful for those of us that stagger/test our updates before rolling them out to live sites.

All other EDD plugins still have dates attached, so I'm assuming this is just a mistake/oversight. 

### Actual behavior
You can see the changelog missing dates on the website and in WordPress plugin update. If you scroll to older updates of the plugin, the dates exist.

#### Website changelog
[https://easydigitaldownloads.com/downloads/stripe-gateway/](https://easydigitaldownloads.com/downloads/stripe-gateway/)

<img width="943" alt="image" src="https://github.com/awesomemotive/easy-digital-downloads/assets/186682/1f2ab3b4-3a25-45b4-844f-93e474d4d4b2">

#### Plugins page update changelog
<img width="737" alt="image" src="https://github.com/awesomemotive/easy-digital-downloads/assets/186682/58a8923e-89e6-4e14-ab2e-b37bb1e89441">

#### Plugins page

1) Go to plugins update page in WordPress.
2) Click on View Details on the EDD.
3) Click on the Changelog tab. 

#### Website changelog

1) Go to [https://easydigitaldownloads.com/downloads/stripe-gateway/](https://easydigitaldownloads.com/downloads/stripe-gateway/).
2) Click on current version #.

### Information (if a specific version is affected):
PHP Version: 8.1
EDD Version (or branch): Version 3.1.1.4.2
WordPress Version: 6.2.2

**Comment by cklosowski** [comment]
Hey @brianleejackson. The omission of the date is intentional and is becoming part of our process ecosystem wide.

As we've been improving our processes, workflows, and async work environment, we've had multiple situations where having the date in the change log has caused previously approved pull requests to need to be re-approved in order to merge them, and as such further delayed a release.

I do apologize if this does pose challenges in your own processes, but we've made this decision to empower our team to work faster and without roadblocks to our processes at this time.

**Comment by cklosowski** [comment]
@brianleejackson I do want to further mention, that we're still working through some process improvements and automation for our extension releases, so it is possible that dates will return once we've hit full automation of them.

**Comment by brianleejackson** [comment]
@cklosowski Thanks for responding. So how can we currently see when an update is pushed out? Site owners also have to test versions, and when things go wrong, knowing when the update was released is extremely helpful. 

I guess I will have to start documenting the date in Trello when I see a new update available. Which means more work for me.

I've never seen a company remove all dates for updates on site/plugin page in my entire WordPress career. This is a very strange change.

**Comment by cklosowski** [comment]
@brianleejackson When deciding to do this, we looked at some of the top plugins available on .org and about 1/2 of the top 10 don't have dates in their change log, including names like Really Simple SSL, Classic Editor, and All-in-One WP Migration. So some major plugins already don't include these.

As for your concern about when knowing the dates of releases, if it's that important, would it be ideal to have a global EDD changelog list that isn't specific to a single extension or core? 
For example: https://aioseo.com/changelog/

**Comment by brianleejackson** [comment]
Hey @cklosowski, 

Changelogs are very simple. You should have two things. 

1. What was changed. 
2. When it was changed. 

Just because some of the most popular plugins on the repository started removing dates doesn't mean that's a good thing to follow. I don't use any of those plugins.

If you don't want to include them in the plugin changelog, why not keep them on the site changelog for the specific extension? That keeps anything out of your plugin code commits. Why remove them from both the code and the site?

**Comment by cklosowski** [comment]
> Why remove them from both the code and the site

This is a case of we removed it from code, which then removed it from the site. They were all done via the same build process and procedures we had been using.

We are going to be introducing a full changelog that is available via RSS in the near future, much like I linked above from AIOSEO, but I'm still working on building out the back log of all our releases to get it live.

**Comment by brianleejackson** [comment]
OK, great! As long as the full changelog will include individual extensions on it, that should work fine. Appreciate the communication.

**Comment by cklosowski** [comment]
The situation here is that our process moved faster than I could implement things on our site to account for this. I made the call to start this process as we needed to get the release out knowing that the full changelog system wasn't ready yet, due to the importance of the release getting out the door.

**Comment by cklosowski** [comment]
@brianleejackson It's a work in progress and this is the first iteration that I had started work on. This custom post type changelog powers all changelogs going forward (once we connect the product to the changelogs). I've gone through and manually moved all releases so far in 2023 into this system, as well as Stripe Pro releases dating back to 2021.

We've got 10 years of changelogs to move over, so this will take some time to be fully implemented, but we'll work on it steady as we go.
